docs/azure/sdk/snippets/protocol-convenience-methods/SCMConvenience/Program.cs

```diff
@@ -1,14 +1,12 @@
-﻿using OpenAI;
-using OpenAI.Chat;
-using System.ClientModel;
+﻿using OpenAI.Chat;
 
 // Create the client
-OpenAIClient client = new("<your-openai-api-key>");
-ChatClient chatClient = client.GetChatClient("gpt-4");
+ChatClient client = new(
+    model: "gpt-4o-mini",
+    credential: Environment.GetEnvironmentVariable("OPENAI_API_KEY")!);
 
 // Call the convenience method
-ChatCompletion completion
-    = chatClient.CompleteChat("What is Microsoft Azure?");
+ChatCompletion completion = client.CompleteChat("What is Microsoft Azure?");
 
 // Display the results
-Console.WriteLine($"{completion.Role}: {completion.Content}");
\ No newline at end of file
+Console.WriteLine($"[{completion.Role}]: {completion}");
```

Comment:
Just to confirm - do we know that `completion` has a `ToString` method implemented that will make the output human-readable?

Reply:
Yes, I tested it locally. The ToString method is implemented, so the output here is human readable.